frontend/src/App.tsx

```diff
@@ -45,6 +72,36 @@ function App() {
     setNewTodoDescription("");
   };
 
+  function deleteTodo(url: string, id: string) {
+    return fetch(url + id, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+  }
+
+  const onDeleteTodo = async (id: string) => {
+    const response = await deleteTodo(urlString, id);
+    const { status } = await response.json();
+    if (status) {
+      setTodos((currentTodo) => currentTodo.filter((todo) => todo.id !== id));
+    }
+  };
+
+  const onChange = (name: string, value: string) => {
+    setNewTodoDescription(value);
+  };
+
+  const content = loading ? (
+    <h3 className="msg-text">LOADING...</h3>
+  ) : !loading && todos.length !== 0 ? (
+    <TodoList todos={todos} deleteTodo={onDeleteTodo}></TodoList>
+  ) : (
+    !loading &&
+    todos.length === 0 && <h3 className="msg-text">NO TASKS ADDED</h3>
```

Comment:
I mentioned this before too, but we are calling them Todos, not Tasks. Change this to "No Todos"